clusterloader2/pkg/util/ssh.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"io"
+	"os/exec"
+
+	"k8s.io/api/core/v1"
+	"k8s.io/klog"
+)
+
+// SSH executes command on a given node with stdin provided.
+// If stdin is nil, the process reads from null device.
+func SSH(command string, node *v1.Node, stdin io.Reader) error {
```

Comment:
Similar logic/function is used in nodes.go (node-killer), could we change it to use this util?

Can be done as a part of separate PR